torch_geometric/datasets/__init__.py

```diff
@@ -138,6 +140,8 @@
     'BAShapes',
     'MalNetTiny',
     'OMDB',
+    'Polblogs',
+    'Email_eu_core',
```

Comment:
```suggestion
    'EmailEUCore',
```
I think we do not want to use snake-case in class names. Prefer `EmailEUCore`.